### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by vidhi-thakur**
- created a simple footer
image

- updated primary color

### CODE REVIEW

**Review by maitrakhatri — APPROVED** [review]
Code looks good.
Liked that you have used variables for spacing

**Review by NeerajVodala — APPROVED** [review]
Everything looks good.
Liked how you used variables for spacing too.

File: components/component_style/footer.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.footer {
+  background-color: var(--primary-color);
+  color: var(--basic-bg);
+  padding: var(--default-spacing) 0rem;
```
Comment (msdeshmukh009) [inline components/component_style/footer.css]: You don't need to specify unit for 0.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

File: components/component_style/footer.css (hunk 1 above)
Comment (vidhi-thakur) [reply components/component_style/footer.css]: thanks for review, have made the change

**Comment by vidhi-thakur** [comment]
> Code looks good. Liked that you have used variables for spacing

thanks for the review

**Comment by vidhi-thakur** [comment]
> Everything looks good. Liked how you used variables for spacing too.

thanks for the review

**Pull request merged**